docs/launchpad/guides/postgresql_ha.md

```diff
@@ -82,3 +122,214 @@ graph-database:
                   - name: BACKUP_SCHEDULE
                     value: '00 00 * * sun'
 ```
+
+:::critical
+Important mentions about WAL files that can impact the availability of your cluster:
+
+- Should a replica experience a failure, it's important to note that WAL files will be retained and not deleted until either the replica has been successfully recovered, or removed. This retention policy is crucial for ensuring data integrity and consistency across the database cluster. However, it can lead to rapid disk space consumption, posing a risk of exhausting available storage.
+
+- Whenever taking a basebackup (new replica, standby, etc), WAL files accumulate at a fast pace and exhaust disk space. Beware of that.
+:::
+
+## Setting up Clones and Standby Clusters
+
+A Clone is a PIT copy of the production database that one would use for testing for development or for major upgrades - think of it as an independent staging area. The postgres-operator allows for two ways to create clones:
+
+- Clone from an S3 bucket (recommended)
+- Clone directly from a live instance
+
+To clone from an S3 bucket you need to define a new PostgreSQL CRD resource with `spec.clone` section defined. Example:
+
+```yaml
+apiVersion: "acid.zalan.do/v1"
+kind: postgresql
+metadata:
+  name: primary-subgraph-data-clone
+spec:
+  clone:
```

Comment:
Maye we want to comment on this section:
- that some of those are optional
- that the default s3_wal_path follows s3://<bucketname>/spilo/<source_db_cluster>/<UID>/wal/<PGVERSION>
- not sure if it applies to clone, but at least in standby's using live instance (cluster, which would be standby_host in that case) is mutually exclusive with using an s3_bucket (so you specify either s3_wal_path or standby_host, but both errors out)

Reply:
- added comments in the code for what's optional
- so cluster is required for clones at all times at least according to the docss3://custom/path/to/bucket . Re s3_wal_path not sure if that's correct for clones (following the user docs) but I have updated it on the standby section.
- Re mutual exclusivity it only applies for standbys and I've added a note about the fact it errors out on line 220

Reply:
it's the same bucket be it a clone or a standby so, the same path :p